src/django_peerctl/templates/peerctl/v2/base.html

```diff
@@ -155,7 +155,7 @@
   <div class="col-4">
     <div class="alert alert-warning marg-top-lg">
 {% blocktrans trimmed with app_id=request.app_id org_name=request.org.display_name_verbose %}
-{{ org_name }} is currently not permissioned to manage any ASNs. Please contact fullCtl support for assistance.
+{{ org_name }} is currently not permissioned to manage any ASNs. Please contact <a href="mailto:support@fullctl.com">support@fullctl.com</a> support for assistance or <a href="https://docs.fullctl.com/aaaCtl/Sign-up-for-PeeringDB/">connect your account to PeeringDB</a>.
```

Comment:
@jason20c email address should come from {{ support_email }} -> https://github.com/fullctl/fullctl/blob/main/src/fullctl/django/context_processors.py#L13